src/core/Form/TextInput/TextInput.baseStyles.tsx

```diff
@@ -69,9 +69,12 @@ export const baseStyles = withSuomifiTheme(
       width: 100%;
       min-height: 40px;
       padding-left: ${theme.spacing.insetL};
+      border-color: ${theme.colors.depthDark3};
```

Comment:
Is this the only place where border-color has changed? What about SearchInput, checkbox and radiobutton?

Reply:
It is changed for them as well.

Reply:
So it was, don't know how I missed it!